paper/paper.md

```diff
@@ -99,7 +122,8 @@ storage of loaded data on the client side, and WebAssembly has been designed to
 interface well with JavaScript. Further, the primitive structure of WebAssembly
 reduces the time to calculate the image in the browser. Finally, WebAssembly
 is executed in a sandboxed environment separate from other processes 
-and is memory safe. 
+and is memory safe. At the time of writing, widgyts is the only
+webassembly-backed visualization widget in the python ecosystem. 
```

Comment:
I'm not sure this remains true -- I believe that itk is built on compiled-to-wasm code.  You might tweak it to say, "webassembly-native" or something to indicate that it was originally written for this purpose.

Reply:
Ok I've modified the sentence slightly! Let me know if that seems good to you!